src/SmartComponents/Inventory/Inventory.js

```diff
@@ -41,7 +41,7 @@ class InventoryTable extends Component {
             } }>
                 <Card>
                     <CardHeader>
-                        <Filter { ...props } filters={ filters } pathPrefix={ pathPrefix } apiBase={ apiBase } />
+                        <Filter { ...props } items={ items } filters={ filters } pathPrefix={ pathPrefix } apiBase={ apiBase } />
```

Comment:
So if we're dealing with really large lists of items, maybe this should be a flag/bool rather than passing that really large list yet again somewhere in the app?

Reply:
Good point, gonna add `hasItems` prop.